app/javascript/dashboard/routes/dashboard/notifications/components/NotificationTable.vue

```diff
@@ -15,7 +15,7 @@
           :key="notificationItem.id"
           @click="() => onClickNotification(notificationItem)"
         >
-          <td>
+          <td class="text-truncate">
             <div class="flex-view notification-contant--wrap">
```

Comment:
@iamsivin On testing, it seems that it is better to put text-truncate here.

Reply:
Updated the PR